appveyor.yml

```diff
@@ -1,3 +1,6 @@
+image:
+  - Visual Studio 2019
+
 environment:
```

Comment:
You could set the version like [this](https://github.com/autofac/Autofac.Extensions.DependencyInjection/blob/develop/appveyor.yml#L1) instead. we would require a version `5.0.0.{Build}` since there are breaking changes involved.